packages/firebase_vertexai/firebase_vertexai/lib/src/live_api.dart

```diff
@@ -0,0 +1,302 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import 'api.dart';
+import 'content.dart';
+import 'error.dart';
+
+/// Represents the available voice options for speech synthesis.
+enum Voices {
+  // ignore: public_member_api_docs, constant_identifier_names
```

Comment:
I'm not sure it makes sense to violate `constant_identifier_names` here. Was there a cross-SDK decision to capitalize this way?

Reply:
No it's not cross-SDK decision yet. However those feels character name to me, that's why I think it needs to be capitalized.

Reply:
I don't think it's a good idea to capitalize them. Dart conventions would use lowercase enum names even for proper nouns.

Reply:
Ok, I will put those into lower cases